test/container-structure/kuma-init.yaml

```diff
@@ -10,10 +10,13 @@ commandTests:
 - name: "Contains kumactl"
   command: kumactl
   args: ["version"]
-- name: "Contains iptables"
-  command: iptables
-  args: ["--version"]
-  expectedOutput: ["iptables v.*"]
+# Skipping iptables test as it fails on arm64 with qemu-user-static
```

Comment:
Normally I'd suggest skipping only on arm64 but if iptables is not present at all then other tests will fail so 👍